### PR TITLE
Documentation: Start Page and Quickstart changes

### PR DESCRIPTION
**Pull request by tsavina**
Link to a test build: http://openvino-doc.iotg.sclab.intel.com/model_server_start_page/ovms_what_is_openvino_model_server.html

### CODE REVIEW

File: docs/ovms_quickstart.md [hunk 1]
```diff
@@ -16,91 +18,82 @@ To quickly start using OpenVINO™ Model Server follow these steps:
...
+- [Serving models](starting_server.md)
+- [Serving multiple model versions](model_version_policy.md) 
+
+### Step 4: Start the Model Server Container
 
+Start the container:
+
+```bash
+docker run -d -u $(id -u):$(id -g) -v $(pwd)/model:/models/face-detection -p 9000:9000 openvino/model_server:latest \
+--model_path /models/face-detection --model_name face-detection --port 9000 --plugin_config '{"CPU_THROUGHPUT_STREAMS": "1"}' --shape auto
```
Comment (dtrawins) [inline docs/ovms_quickstart.md:67]:
```suggestion
--model_path /models/face-detection --model_name face-detection --port 9000
```

File: docs/ovms_quickstart.md [hunk 2]
```diff
@@ -16,91 +18,82 @@ To quickly start using OpenVINO™ Model Server follow these steps:
...
+```
+During this step, the model folder is mapped to Docker. This folder will be used as the model storage from which the server will access models.
```
Comment (dtrawins) [inline docs/ovms_quickstart.md:69]:
```suggestion
During this step, the `model` folder is mounted to the Docker container. This folder will be used as the model storage from which the server will access models.
```

File: docs/ovms_quickstart.md [hunk 3]
```diff
@@ -16,91 +18,82 @@ To quickly start using OpenVINO™ Model Server follow these steps:
...
 
 ### Step 5: Prepare the Example Client Components
 
-Model scripts are available to provide an easy way to access Model Server. Here is an example command, using face detection and curl, to download all necessary components:
+Model scripts are available for quick access to the Model Server. Run an example command to download all required components:
```
Comment (dtrawins) [inline docs/ovms_quickstart.md:73]:
```suggestion
Client scripts are available for quick access to the Model Server. Run an example command to download all required components:
```

**Review by dtrawins — APPROVED** [review]

File: docs/ovms_quickstart.md [hunk 4]
```diff
@@ -16,91 +18,82 @@ To quickly start using OpenVINO™ Model Server follow these steps:
...
+- [Serving models](starting_server.md)
+- [Serving multiple model versions](model_version_policy.md) 
+
+### Step 4: Start the Model Server Container
 
+Start the container:
+
+```bash
+docker run -d -u $(id -u):$(id -g) -v $(pwd)/model:/models/face-detection -p 9000:9000 openvino/model_server:latest \
+--model_path /models/face-detection --model_name face-detection --port 9000
```
Comment (ngrozae) [inline docs/ovms_quickstart.md:67]:
```suggestion
--model_path /models/face-detection --model_name face-detection --port 9000 --shape auto
```
without shape auto parameter " python face_detection.py " will fail

**Review by mzegla — APPROVED** [review]
Looks good. Maybe I come too much from the developer perspective, but maybe we could add a link to GitHub somewhere on top in the home document? What do you think?

**Pull request merged**